.github/workflows/test.yaml

```diff
@@ -1,4 +1,4 @@
-name: E2E test
+name: Tests
 on:
   pull_request:
       branches:
```

Comment:
Just a comment for the future (i totally missed this before)

This will only run on PRs to master, we'll need to add a wildcard for `release-*` to cover forked released branches

Reply:
We can add it now without problems:
```yml
name: Tests
on:
  pull_request:
      branches:
        - master
        - release-*
```

Reply:
go for it